CHANGELOG.md

```diff
@@ -19,14 +19,35 @@
 
 # Table of Contents
 
+- [2.1.1](#211)
 - [2.1.0](#210)
 - [2.0.0](#200)
 - [1.5.0](#150)
 - [1.0.0](#100)
 
+
+# 2.1.1
+
+This release mainly adds test cases and bugfix for `manager api`.
+
+### Test case
+
+Add e2e test cases for route, upstream, consumer, SSL, and plugins.
+
+## Bugfix
+ 
+- Fix: when created route by `admin api`, upstream can not be modified by dashboard #786
```

Comment:
`admin api` to `Admin API`
`manager api` to `Manager API`

I think we should keep the same style.